packages/bpk-component-calendar/src/BpkCalendarDate-test.js

```diff
@@ -73,4 +73,11 @@ describe('BpkCalendarDate', () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  it('should set a custom class', () => {
+    const tree = renderer
+      .create(<BpkCalendarDate date={new Date(2010, 1, 15)} className="fake" />)
```

Comment:
instead of `fake` could you put something like `userlandClassName`.
That way if the snapshot fails, it will be more obvious what has broken

Reply:
Done!